app/controllers/users/registrations_controller.rb

```diff
@@ -75,37 +74,43 @@ def update
   #   super
   # end
 
+  private
```

Comment:
@ruffini47 
```suggestion
  private
  
    def set_user
      @user = User.find(params[:id])
    end
```
ここが鍵です。
only_same_company_id?が@userを使っていました。
前回UsersControllerで使わないことから削除していたのですが
RegistrationsControllerで必要になったのでredirectに引っ掛からなくなってました。

Reply:
「ユーザー情報編集バグ解消」のコミット履歴を見てみると、@userは使わないようにしているようですが。

Reply:
only_same_company_id?メソッドでもともと使っていたので併用するためにここで定義しました。